R/model_eval.R

```diff
@@ -314,15 +314,15 @@ evaluate_multi_ <- function(df, pred_label_col, actual_val_col, pretty.name = FA
 }
 
 #' @export
-evaluate_binary_training_and_test <- function(df, actual_val_col, threshold = "f_score", pretty.name = FALSE, test_rate = 0.0){
+evaluate_binary_training_and_test <- function(df, actual_val_col, threshold = "f_score", pretty.name = FALSE){
```

Comment:
Comment that this is for Summary Table for logistic/binomial regression analytics view.